bento_beacon/config_files/config.py

```diff
@@ -169,6 +169,9 @@ class Config:
     PHENOPACKETS_SCHEMA_REFERENCE = {"entityType": "individual", "schema": "phenopackets v1"}
 
     MAX_RETRIES_FOR_CENSORSHIP_PARAMS = 2
+
+    # don't let anonymous users query arbitrary phenopacket or experiment fields
+    ANONYMOUS_METADATA_QUERY_USES_DISCOVERY_CONFIG_ONLY = True
```

Comment:
technically these aren't anonymous users per se, just ones without `query:data` - could be called `CENSORED_METADATA_QUERY_USES_DISCOVERY_CONFIG_ONLY`

Reply:
(oops)